frontend/src/pages/pipelines/global/modelCustomization/utils.ts

```diff
@@ -163,3 +168,35 @@ export const translateIlabFormToBaseModelInput = (
   sdg_base_model: data.baseModel.sdgBaseModel,
   // TODO more output model fields
 });
+
+export const translateIlabFormToRunType = (
+  data: ModelCustomizationFormData,
+): {
+  runType: string;
+} => ({
+  runType: data.runType.value,
+});
+
+export const translateIlabFormToHyperparameters = (
+  data: ModelCustomizationFormData,
+): {
+  hyperparameters: HyperparametersFormData;
+} => ({
+  hyperparameters: data.hyperparameters,
+});
+
+export const filterHyperparameters = (pipelineVersion: PipelineVersionKF | null): ParametersKF => {
+  let hyperparameters = {};
+  const ilabPipelineParams = getInputDefinitionParams(pipelineVersion);
+  if (typeof ilabPipelineParams !== 'undefined') {
+    for (const key of Object.keys(ilabPipelineParams)) {
+      if (isEnumMember(key, HyperparameterFields)) {
+        hyperparameters = {
+          ...hyperparameters,
+          [key]: ilabPipelineParams[key],
+        };
+      }
+    }
+  }
+  return hyperparameters;
+};
```

Comment:
this is filtering out all except known keys. we should just have an exclusion list which are ones that are done elsewhere and ones that are strictly not included from ux... so its left with known hyperparameters and unknown ones